### PR TITLE
[feature-wip](Cloud) Show isDefault vault in show storage vault stmt

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

```
mysql> show storage vault;
+------------------------+----------------+-------------------------------------------------------------------------------------------------+-----------+
| StorageVaultName       | StorageVaultId | Propeties                                                                                       | IsDefault |
+------------------------+----------------+-------------------------------------------------------------------------------------------------+-----------+
| built_in_storage_vault | 1              | build_conf { fs_name: "hdfs://127.0.0.1:8020" } prefix: "_B50EA87B-456C-310C-C80D-834BC78235E2" | false     |
+------------------------+----------------+-------------------------------------------------------------------------------------------------+-----------+
1 row in set (0.07 sec)

mysql> set built_in_storage_vault as default storage vault;
Query OK, 0 rows affected (0.04 sec)

mysql> show storage vault;
+------------------------+----------------+-------------------------------------------------------------------------------------------------+-----------+
| StorageVaultName       | StorageVaultId | Propeties                                                                                       | IsDefault |
+------------------------+----------------+-------------------------------------------------------------------------------------------------+-----------+
| built_in_storage_vault | 1              | build_conf { fs_name: "hdfs://127.0.0.1:8020" } prefix: "_B50EA87B-456C-310C-C80D-834BC78235E2" | true      |
+------------------------+----------------+-------------------------------------------------------------------------------------------------+-----------+
1 row in set (0.00 sec)
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

File: cloud/src/meta-service/meta_service_resource.cpp [hunk 1]
```diff
@@ -278,6 +278,7 @@ void MetaServiceImpl::get_obj_store_info(google::protobuf::RpcController* contro
     }
 
     response->mutable_obj_info()->CopyFrom(instance.obj_info());
+    response->set_default_storage_vault_id(instance.default_storage_vault_id());
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service_resource.cpp:281]: why not also return default vault name?

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**